kitsune/settings.py

```diff
@@ -752,19 +752,19 @@ def JINJA_CONFIG():
 #
 # Django Pipline
 PIPELINE = {
-    "COMPILERS": ("kitsune.lib.pipeline_compilers.BrowserifyCompiler",),
+    "PIPELINE_ENABLED": config("PIPELINE_ENABLED", default=False, cast=bool),
```

Comment:
`default=not DEBUG` was the value for `"PIPELINE_COLLECTOR_ENABLED"`, I've just copied this from [way lower](https://github.com/mozilla/kitsune/pull/4682/files#diff-8a0565e4a9db6f61b629727419deb11dfc60a56d3215ef360ebf2f7754ec1422L1148) in the settings, and placed it within the PIPELINE dict: https://django-pipeline.readthedocs.io/en/latest/installation.html#upgrading-from-1-5

As for the value of PIPELINE_COLLECTOR_ENABLED, I removed that because we need it to run in dev for our babel transforms to work. Our previous setting didn't make a whole lot of sense, since that setting only has an effect if `PIPELINE_ENABLED` is False: https://django-pipeline.readthedocs.io/en/latest/configuration.html#pipeline-collector-enabled. So the default value was essentially just False.

https://github.com/mozilla/kitsune/commit/3e35508dcaaa6632a25322bc5d7dd5edac0f84b4 was the rationale for including that default value for PIPELINE_COLLECTOR_ENABLED, so since we no longer have any LESS files, I'm confident removing it.